pyaedt/maxwell.py

```diff
@@ -1951,16 +1951,11 @@ def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
         corresponding to the native AEDT API.  The list of
         keywords here is not exhaustive.
 
-        .. note::
-           This method overrides the ``Analysis.setup()`` method for the HFSS app.
-
         Parameters
         ----------
         setuptype : int, str, optional
             Type of the setup. Based on the solution type, options are
-            ``"HFSSDrivenAuto"``, ``"HFSSDrivenDefault"``,
-            ``"HFSSEigen"``, ``"HFSSTransient"`` and ``"HFSSSBR"``.
-            The default is ``"HFSSDrivenAuto"``.
+            "Magnetostatic", "EddyCurrent", Transient" etc.
         setupname : str, optional
             Name of the setup. The default is ``"Setup1"``.
```

Comment:
```suggestion
            Name of the setup. The default is ``"Setup1"``.
```
The default name is not "Setup1"